### PR TITLE
Remove Deprecated OAuthRequestsV1  Functionality

### PR DESCRIPTION
**Pull request by lukemarkwordtlibertyits**
# Description

After 42 Days of Implementing OAuthRequestsV2,  OAuthRequestsV1 can be removed.

# TODO

- [x] Remove code from getDocumentByCodeStrategy
  - [x] Remove `Happy Path - Existing OAuthRequest State` test 
- [x] Remove code from getDocumentByRefreshTokenStrategy
  - [x] change `Could not retrieve Token`
  - [x] Remove  `Could not retrieve Token, ClientID` 
- [x] Remove code from saveDocumentStateStrategy
  - [x] Remove `Happy Path w/o internal_state` test
  - [x] Remove `Happy Path w/o internal_state with launch` Test
  - [x] Remove `Happy Path w/o internal_state err on dynamo save` Test
- [x] Remove code from redirectHandler
  - [x] Remove `Happy Path Redirect with OAuthRequests state` Test
- [x] Remove OAuthRequests from CLI
- [x] Remove OAuthRequests from dev-config.json
- [x] Remove OAuthRequests from createFakeConfig method in testUtils.js

# Tests

- [x] Regression Tests
  - [x] Code Coverage Should be met 
    - Met at 90.94%
## Backwards compatibility tests should no longer work

**Steps**

- Go through code flow, grab refresh token.
- Inject OAuthRequestsV2 values into OAuthRequests
- Clear OAuthRequestsV2 table
- Go through refresh flow 

**Before Changes**

- [x] Should respond with a Token Object, with state.

**After Changes**

- [x] Should respond with a Token Object, w/o state.

### CODE REVIEW

File: oauth-proxy/oauthHandlers/redirectHandler.js [hunk 1]
```diff
@@ -32,7 +31,7 @@ const redirectHandler = async (
   } catch (error) {
     logger.error("Failed to look up by internal_state.", error);
   }
-
+  // REMOVE
```
Comment (beaugrantham-va) [inline oauth-proxy/oauthHandlers/redirectHandler.js:34]: Is this PR ready?

Comment (lukemarkwordtlibertyits) [reply oauth-proxy/oauthHandlers/redirectHandler.js:34]: My bad

File: oauth-proxy/oauthHandlers/tokenHandlerStrategyClasses/documentStrategies/getDocumentByRefreshTokenStrategy.js [hunk 2]
```diff
@@ -21,34 +17,6 @@ class GetDocumentByRefreshTokenStrategy {
...
-      );
-      document = await this.getDocumentDynamo(
-        hashedRefreshToken,
-        this.config.dynamo_table_name
-      );
-    }
-
-    // Backwards compatibility.
-    // Remove after 42 days of PR merge (DATE - 02/23/2021).
-    if (!document) {
```
Comment (beaugrantham-va) [inline oauth-proxy/oauthHandlers/tokenHandlerStrategyClasses/documentStrategies/getDocumentByRefreshTokenStrategy.js:38]: This block of code belongs with the deferred part of the story (rejecting stateless tokens).

I think this should remain in place until we get to that second story. It allows us to have continued visibility into these tokens for a bit longer.

**Review by beaugrantham-va — CHANGES_REQUESTED** [review]

**Comment by lukemarkwordtlibertyits** [comment]
I have confirmed that the refresh_token retrieval warning is working after adding it back in

image

**Review by beaugrantham-va — APPROVED** [review]

**Review by bdgrider-va — APPROVED** [review]
Verified tests pass.

**Comment by beaugrantham-va** [comment]
@lukemarkwordtlibertyits Please open a PR to remove the deprecated config from the real config files.

**Pull request merged**